packages/frontend/editor-ui/src/components/Node/NodeCreator/NodeCreator.vue

```diff
@@ -153,7 +159,13 @@ onBeforeUnmount(() => {
 	unBindOnMouseUpOutside();
 });
 
-onClickOutside(nodeCreator, () => emit('closeNodeCreator'));
+onClickOutside(
+	nodeCreator,
+	() => {
+		emit('closeNodeCreator');
+	},
+	{ ignore: OUTSIDE_CLICK_WHITELIST },
```

Comment:
Would be better if the overlay didn't allow the click event to bubble up. But if it's hard don't waste too much time on it, this is fine.